### PR TITLE
(fr) Broken links

### PR DESCRIPTION
**Pull request by butuzov**
Multiple broken links

* malfomed urls in glossary (had trailing "fr/")
* malformed url docs (no trailing "/")
* relative link ".."

### CODE REVIEW

File: content/fr/docs/reference/glossary/pod.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Pod
 id: pod
 date: 2018-04-12
-full_link: fr/docs/concepts/workloads/pods/pod-overview/
+full_link: /docs/concepts/workloads/pods/pod-overview/
```
Comment (sftim) [inline content/fr/docs/reference/glossary/pod.md]: (nit) ou
```suggestion
full_link: /fr/docs/concepts/workloads/pods/pod-overview/
```

File: content/fr/docs/reference/_index.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Pour appeler l'API de Kubernetes depuis un langage de programmation on peut util
 
 ## Documents de Référence des outils en ligne de commande (CLI)
 
-* [kubectl](docs/reference/command-line-tools-reference/kubelet/) - Principal outil en ligne de commande (CLI) pour exécuter et gérer un cluster Kubernetes.
+* [kubectl](/docs/reference/command-line-tools-reference/kubelet/) - Principal outil en ligne de commande (CLI) pour exécuter et gérer un cluster Kubernetes.
```
Comment (oussemos) [inline content/fr/docs/reference/_index.md]: it should redirect to /docs/reference/kubectl/overview/ instead (see the english version https://kubernetes.io/docs/reference/)

File: content/fr/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ card:
 
 {{% capture overview %}}
 
-Ce tutoriel vous montre comment exécuter une simple application Hello World Node.js sur Kubernetes en utilisant [Minikube](/docs/getting-start-guides/minikube) et Katacoda.
+Ce tutoriel vous montre comment exécuter une simple application Hello World Node.js sur Kubernetes en utilisant [Minikube](/docs/getting-started-guides/minikube) et Katacoda.
```
Comment (sftim) [inline content/fr/docs/tutorials/hello-minikube.md]:
```suggestion
Ce tutoriel vous montre comment exécuter une simple application Hello World Node.js sur Kubernetes en utilisant [Minikube](/docs/getting-started-guides/minikube/) et Katacoda.
```

Comment (butuzov) [reply content/fr/docs/tutorials/hello-minikube.md]: IMHO, no need, as such links will be handled by server.

File: content/fr/docs/reference/_index.md (hunk 2 above)
Comment (butuzov) [reply content/fr/docs/reference/_index.md]: Hm, miss it.

**Pull request merged**